### ISSUE TITLE
on replay, monitor if GPU hangs

### ISSUE THREAD

**Issue by okias** [apitrace/apitrace#787]
There should be possibility to monitor GPU hangs. Should be useful for the `replay` in CI environment to report it to developers together with the failure itself.

**Comment by jrfonseca** [comment]
I'm not sure I understand.  Many GPU hangs lead to system hangs, but I suppose you mean the other kind of GPU hangs -- non- fatal ones.  You mean glretrace could scavenge `dmesg` etc for signs of GPU resets?

**Comment by okias** [comment]
exactly, only hangs where GPU is able to recover. The scavenging of `dmesg` could be done outside apitrace, just signal it's needed would be great.

In CI we record the `dmesg` but this could nicely point directly to the part, where failure happens.

I added link to Mozilla discussion about this topic.
